Install/update_gopigo3.sh

```diff
@@ -115,40 +119,45 @@ parse_cmdline_arguments() {
-    command -v git >/dev/null 2>&1 || { echo "This script requires \"git\" but it's not installed. Don't use --no-dependencies option. Exiting." >&2; exit 1; }
-    command -v python >/dev/null 2>&1 || { echo "Executable \"python\" couldn't be found. Don't use --no-dependencies option. Exiting." >&2; exit 2; }
-    command -v python3 >/dev/null 2>&1 || { echo "Executable \"python3\" couldn't be found. Don't use --no-dependencies option. Exiting." >&2; exit 3; }
-    command -v pip >/dev/null 2>&1 || { echo "Executable \"pip\" couldn't be found. Don't use --no-dependencies option. Exiting." >&2; exit 4; }
-    command -v pip3 >/dev/null 2>&1 || { echo "Executable \"pip3\" couldn't be found. Don't use --no-dependencies option. Exiting." >&2; exit 5; }
-  fi
-
   # create rest of list of arguments for rfrtools call
   rfrtools_options+=("$selectedbranch")
   [[ $usepython3exec = "true" ]] && rfrtools_options+=("--use-python3-exe-too")
   [[ $updaterepo = "true" ]] && rfrtools_options+=("--update-aptget")
   [[ $installdependencies = "true" ]] && rfrtools_options+=("--install-deb-deps")
   [[ $install_pkg_rfrtools = "true" ]] && rfrtools_options+=("--install-python-package")
-
-  # create list of arguments for script_tools call
-  declare -ga scriptools_options=("$selectedbranch")
+  [[ $install_rfrtools_gui = "true" ]] && rfrtools_options+=("--install-gui")
 
   echo "Using \"$selectedbranch\" branch"
   echo "Options used for RFR_Tools script: \"${rfrtools_options[@]}\""
-  echo "Options used for script_tools script: \"${scriptools_options[@]}\""
 }
 
 #################################################
-## Cloning GoPiGo3, Script_Tools & RFR_Tools ####
+########## Cloning GoPiGo3 & RFR_Tools ##########
 #################################################
 
+# called in <<install_rfrtools_repo>>
+check_dependencies() {
+  command -v git >/dev/null 2>&1 || { echo "This script requires \"git\" but it's not installed. Error occurred with RFR_Tools installation." >&2; exit 1; }
+  command -v python >/dev/null 2>&1 || { echo "Executable \"python\" couldn't be found. Error occurred with RFR_Tools installation." >&2; exit 2; }
+  command -v pip >/dev/null 2>&1 || { echo "Executable \"pip\" couldn't be found. Error occurred with RFR_Tools installation." >&2; exit 3; }
+  if [[ $usepython3exec = "true" ]]; then
+    command -v python3 >/dev/null 2>&1 || { echo "Executable \"python3\" couldn't be found. Error occurred with RFR_Tools installation." >&2; exit 4; }
+    command -v pip3 >/dev/null 2>&1 || { echo "Executable \"pip3\" couldn't be found. Error occurred with RFR_Tools installation." >&2; exit 5; }
+  fi
+
+  if [[ ! -f $DEXTERSCRIPT/functions_library.sh ]]; then
+    echo "script_tools didn\'t get installed. Enable the installation of dependencies with RFR_Tools.'"
+    exit 8
+  fi
+}
+
 # called way down below
-install_scriptools_and_rfrtools() {
+install_rfrtools_repo() {
```

Comment:
remind me what happened to script_tools? I know we discussed it but I forgot.  Which would indicate the need for a comment :)

Reply:
`script_tools` always gets installed with `RFR_Tools` and every robot installs `RFR_Tools`, which means every robot gets `script_tools` installed. Currently, when a robot is installed, script_tools is installed twice and that's not efficient. 

On top of that, we want to centralize our install scripts and to do that, we need to put more emphasis on RFR_Tools and not the robots' scripts. One of the first steps is to let RFR_Tools install script_tools and not depend on a given robot to install it.